utils/scripts/install-el9.sh

```diff
@@ -56,8 +58,16 @@ dnf --nodocs install \
     valgrind-devel \
     which \
     yasm
+
 # No packages for the one below have been
 # identified yet. Limit build to client only for now
 #    ipmctl \
 #    libipmctl-devel \
 #    Lmod \
+#
+# ipmctl is only available on x86_64
+if [ "$arch" = x86_64 ]; then
```

Comment:
(lint) The mentioned syntax error was in this if expression. [SC1009]
(lint) Couldn't parse this then clause. Fix to allow more checks. [SC1073]